### DISCUSSION TITLE
Cant get data of page

### DISCUSSION THREAD

**Discussion by talhag3** [kevinzg/facebook-scraper#561] [Q&A] [answered]
facebook.com/t20WorldCup

**Comment by neon-ninja** [comment, marked as answer]
Try this commit - https://github.com/kevinzg/facebook-scraper/commit/5f95d80ba9689bf4d323939ecfcf0602c9173b35 - with this commit, the code: `pprint(get_page_info("T20WorldCup", cookies="cookies.json"))` outputs
```python
{'about': 'Category\n'
          'Sports\n'
          'Contact Info\n'
          '/T20WorldCup\n'
          'Facebook\n'
          'http://www.t20worldcup.com/\n'
          'Websites\n'
          'Basic Info\n'
          'Birthday\n'
          'Page Transparency\n'
          'See all\n'
          'Facebook is showing information to help you understand the purpose '
          'of this Page.\n'
          'April 20, 2018\n'
          'Page Creation Date\n'
          'Page Admin Info\n'
          'This Page can have multiple admins. They may have permission to '
          'post content, comment or send messages as the Page.\n'
          'This Page is currently running ads.\n'
          'Life Events'}
```

**Reply by talhag3** [reply]
it works fine with cookies file 👍 
need help for dev setup I'm new to python

**Reply by neon-ninja** [reply]
What do you mean by that? All you need to develop Python is a text editor (I use Visual Studio Code) and a terminal (I use Windows Terminal with Windows Subsystem for Linux)

**Reply by talhag3** [reply]
im setting-up this project for dev or contribute . i create virtual env , install dev dependencies . 
open in vs-code 
how to run it for test changes   

![image](https://user-images.githubusercontent.com/16940319/142846753-77c11365-e52e-4226-b08d-8bea52961d87.png)

![image](https://user-images.githubusercontent.com/16940319/142846834-4d83fafa-9b2a-4308-82a7-b54fbde4e9ca.png)


**HOW TO RUN IT IN DEV**

i know this is very annoying , but im noob .

**Reply by neon-ninja** [reply]
It depends - do you want to:

1. Use the CLI
2. Use the scripting interface
3. Or run the tests?
